### PR TITLE
chore(cypress): fix tests

### PR DESCRIPTION
**Pull request by hgw77**
# Summary

* this will fix some failing tests

# Related Issues

#1446 

- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have made corresponding changes to the documentation (if applicable).
- [x] My changes generate no new warnings or errors.

### CODE REVIEW

File: e2e/cypress/integration/admin/domain.js [hunk 1]
```diff
@@ -23,8 +19,7 @@ describe("domain landing page", () => {
     // eslint-disable-next-line cypress/no-unnecessary-waiting
     cy.wait(500)
     // check not in one string because it can be different order
-    cy.contains("SAP Converged Cloud")
-    cy.contains("button", "Log in")
+    cy.contains("button", "Enter CC3TEST")
```
Comment (ArtieReus) [inline e2e/cypress/integration/admin/domain.js:22]: Would be better hier to check for an fix button.id instead with the button text? If we change the button text the test would then fail...

File: e2e/cypress/integration/member/dns.js [hunk 2]
```diff
@@ -22,14 +18,12 @@ describe("dns", () => {
     cy.contains("Request New Domain")
     cy.get("#zone_request_domain_pool").select("Internal SAP Hosted Zone")
     // click Subdomain
-    cy.get("input#zone_request_domain_type_subdomain")
-      .should("be.visible")
-      .click()
-    cy.get("input#zone_request_name").should("be.visible")
+    //cy.get("input#zone_request_domain_type_subdomain")
```
Comment (ArtieReus) [inline e2e/cypress/integration/member/dns.js:21]: We might habe to remove commented code

**Review by ArtieReus — COMMENTED** [review]
This pull request looks good, but wouldn't be a good chance to improve it using element ID instead of generally looking for button text to check or do an action?

File: e2e/cypress/integration/admin/domain.js (hunk 1 above)
Comment (hgw77) [reply e2e/cypress/integration/admin/domain.js:22]: this is on purpose to check that the automatic generation of the button title is working, the test is running on `CC3TEST` domain so we should see the button with this text

Comment (ArtieReus) [reply e2e/cypress/integration/admin/domain.js:22]: I was meaning generally in the tests. But in this specific one I would do following:

```js
cy.get("#yourButtonID").should("have.text", "Enter CC3TEST");
```

**Review by andypf — APPROVED** [review]
LGTM

**Pull request merged**